File: app/src/main/java/com/victoria/bleled/data/remote/response/ResponseSearchRepo.java
package com.victoria.bleled.data.remote.response;


import com.victoria.bleled.data.model.BaseModel;
import com.victoria.bleled.data.model.ModelUser;

import java.util.List;

public class ResponseSearchRepo extends BaseModel {
    public int total = 0;
    public List<ModelUser> items = null;
}

File: app/src/main/java/com/victoria/bleled/data/remote/github/ResponseSearchRepo.kt
package com.victoria.bleled.data.remote.github

import com.victoria.bleled.data.model.BaseModel
import com.victoria.bleled.data.model.ModelUser

class ResponseSearchRepo : BaseModel() {
    var total = 0

    @JvmField
    var items: List<ModelUser>? = null
}